libs/core/src/components/pds-popover/pds-popover.tsx

```diff
@@ -0,0 +1,162 @@
+import { Component, Element, Event, Host, Method, Prop, State, h, EventEmitter, Listen } from '@stencil/core';
+import {
+  positionTooltip
+} from '../../utils/overlay';
+
+/**
+ * @slot trigger - The popover's target element
+ * @slot content - HTML content for the popover
+ */
+
+@Component({
+  tag: 'pds-popover',
+  styleUrls: ['../../global/styles/base.scss', 'pds-popover.scss'],
+  shadow: true,
+})
+export class PdsPopover {
+  private contentEl: HTMLElement | null;
+  /**
+   * Reference to the Host element
+   */
+  @Element() el: HTMLPdsPopoverElement;
+
+  /**
+   * Determines when the popover is open
+   * @defaultValue false
+   */
+  @State() isOpen = false;
+
+  /**
+   * A unique identifier used for the underlying component `id` attribute.
+   */
+  @Prop() componentId: string;
+
+  /**
+   * Determines whether or not the popover has an arrow
+   * @defaultValue true
+   */
+  @Prop() hasArrow? = true;
+
+  /**
+   * Determines the preferred position of the popover
+   * @defaultValue "right"
+   */
+  @Prop({ reflect: true }) placement:
+    'top'
+    | 'top-start'
+    | 'top-end'
+    | 'right'
+    | 'right-start'
+    | 'right-end'
+    | 'bottom'
+    | 'bottom-start'
+    | 'bottom-end'
+    | 'left'
+    | 'left-start'
+    | 'left-end' = 'right';
```

Comment:
Can we extract this out to a custom type? e.g `type PlacementType` this could then be used for both ToolTip and PopOver